### PR TITLE
[1249 Meta] UI Broken in IE9

### PR DESCRIPTION
**Pull request by stanica**
Fixing issues in IE9

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @stanica received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e037a5aa02937b0/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e037a5aa02937b0/output.txt

Total script time: 0.36 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @stanica received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/1d27ca6203db6b1/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/1d27ca6203db6b1/output.txt

Total script time: 1.16 mins

Server is running on http://pr682.butyr.org/templates/test.html

File: cornfield/app.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ var mongoose = require('mongoose'),
     }),
     UserModel = mongoose.model( 'User', User );
 
-CONFIG.session.store = new MongoStore({ db: "test" });
+//CONFIG.session.store = new MongoStore({ db: "test" });
```
Comment (mjschranz) [inline cornfield/app.js:62]: Is there an issue somehow with Mongo and IE9?

File: src/editor/trackevent-editor.js [hunk 2]
```diff
@@ -221,8 +221,12 @@ define( [ "util/lang", "util/keys", "./base-editor",
...
           editorElement.value = data;
         }
-        editorElement.type = manifestEntry.type;
-
+        try {
+          editorElement.type = manifestEntry.type;
+        }
+        catch (e) {
+          editorElement.type = "text";
+        }
```
Comment (mjschranz) [inline src/editor/trackevent-editor.js:229]: How could assigning editorElement.type to manifestEntry.type every throw an error? Can you explain what was going on here?

File: cornfield/app.js (hunk 1 above)
Comment (stanica) [inline cornfield/app.js:62]: No this was me disabling Mongo and forgetting to enable it! My bad.

File: src/editor/trackevent-editor.js (hunk 2 above)
Comment (stanica) [inline src/editor/trackevent-editor.js:229]: When manifestEntry.type == "number" and editorElement.type == "text", IE9 doesn't automatically convert number to text.

**Comment by jbuck** [comment]
wont-fixed in Lighthouse

**Pull request closed without merging**